### ISSUE TITLE
integration: docker attach through websocket failed

### ISSUE THREAD

**Issue by chanwit** [docker/swarm#701]
The test case "docker attach through websocket" failed on my machine. How should I setup to let the test go through?

```
 ✗ docker attach through websocket
   (in test file test/integration/api.bats, line 44)
     `[ "${#lines[@]}" -eq 2 ]' failed
   time="2015-05-03T18:17:10+07:00" level=info msg="Listening for HTTP" addr="127.0.0.1:6202" proto=tcp
   time="2015-05-03T18:17:10+07:00" level=info msg="HTTP request received" method=GET uri="/v1.18/info"
   time="2015-05-03T18:17:10+07:00" level=info msg="HTTP request received" method=POST uri="/v1.18/containers/create?name=test_container"
   time="2015-05-03T18:17:27+07:00" level=info msg="HTTP request received" method=POST uri="/v1.18/containers/660c665fa5e987b5ac13362deab972af4a9c1c75ba91d6432bde1b93ec413e88/start"
   time="2015-05-03T18:19:45+07:00" level=info msg="HTTP request received" method=GET uri="/v1.17/containers/test_container/attach/ws?stderr=1"
   Stopping 033a60ac236a45ad131a80e731c0e4943c5fc8576ea3b88ea60c03d1264cd5ac
   Stopping 0c839ecfbbea4ce258211afa3f54ab4732b37006bfe3403996c0e83d1e73c68e
```

**Comment by aluzzardi** [comment]
It failed on my machine as well.

/cc @jimmyxian

**Comment by jimmyxian** [comment]
I test locally, it's OK. can you provide some information about your environment so that i can reproduce it.

**Comment by aluzzardi** [comment]
Once we merge #696 we'll share the same environment so it'll be easier to reproduce.

**Comment by jimmyxian** [comment]
@aluzzardi OK

**Comment by aluzzardi** [comment]
@jimmyxian 

Can you try with:

`test/integration/run.sh api.bats`

**Comment by jimmyxian** [comment]
@aluzzardi OK, i will try it

**Comment by jimmyxian** [comment]
I got it.
If the image jimmyxian/cenos7-wssh is not in local, the output of `run docker run --rm --net=host jimmyxian/centos7-wssh wssh $URL` will be as the following. It contains not only `Session Open Session Closed` but also some other information 

```
time="2015-05-04T16:15:07Z" level=info msg="HTTP request received" method=GET uri="/v1.17/containers/test_container/attach/ws?stderr=1" 
Unable to find image 'jimmyxian/centos7-wssh:latest' locally Pulling repository jimmyxian/centos7-wssh dc9c162ff812: Pulling image (latest) from jimmyxian/centos7-wssh dc9c162ff812: Pulling image (latest) from jimmyxian/centos7-wssh, endpoint: https://registry-1.docker.io/v1/ dc9c162ff812: Pulling dependent layers 511136ea3c5a: Download complete b6718650e87e: Download complete 41054cbffecc: Download complete 214a4932132a: Download complete 57afa14d5351: Download complete dc9c162ff812: Download complete dc9c162ff812: Download complete Status: Image is up to date for jimmyxian/centos7-wssh:latest Session Open Session Closed
```

**Comment by chanwit** [comment]
@jimmyxian @aluzzardi shall I close this one?

**Comment by jimmyxian** [comment]
@chanwit I think so :)

**Comment by aluzzardi** [comment]
Fixed by #720
